plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -80,11 +81,11 @@ func (s *SchedulerServer) Run(_ []string) error {
 
 	go func() {
 		if s.EnableProfiling {
-			mux := http.NewServeMux()
-			mux.HandleFunc("/debug/pprof/", pprof.Index)
-			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
-			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+			http.HandleFunc("/debug/pprof/", pprof.Index)
+			http.HandleFunc("/debug/pprof/profile", pprof.Profile)
+			http.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
```

Comment:
I've just faced the same. The issue is:

    panic: http: multiple registrations for /debug/pprof/
goroutine 11 [running]:
net/http.(*ServeMux).Handle(0xc20800a8a0, 0x9e9490, 0xd, 0x7f7d12f190a0, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1557 +0x244
net/http.(*ServeMux).HandleFunc(0xc20800a8a0, 0x9e9490, 0xd, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1585 +0x6d
net/http.HandleFunc(0x9e9490, 0xd, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1597 +0x48
github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app.func·001()
        /go/src/github.com/GoogleCloudPlatform/kubernetes/_output/dockerized/go/src/github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app/server.go:84 +0x58
created by github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app.(*SchedulerServer).Run
        /go/src/github.com/GoogleCloudPlatform/kubernetes/_output/dockerized/go/src/github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app/server.go:90 +0x1c7